org-formation/800-redirects/_tasks.yaml

```diff
@@ -185,7 +185,7 @@ SchematicProdAppDnsForward:
     # ID of the api.sagebionetworks.org zone (in sageit account)
```

Comment:
Could you update the comment on L173 to be 
`https://github.com/Sage-Bionetworks-IT/schematic-infra-v2`

The best looks good